src/Application/Components/Shop/KeyBoundShopItem.jsx

```diff
@@ -16,6 +18,10 @@ class KeyBoundShopItem extends React.Component {
 
   render() {
     const imgSrc = `images/${this.props.name}.png`;
+    let itemDetails = false;
+    if (this.props.areStatsVisible) {
+      itemDetails = <ShopItemStats key={this.props.name} name={this.props.name} />;
```

Comment:
when using the same prop (or just accessing `this.props` a bunch) u can use this notation ```const { areStatsVisible, name, owned } = this.props